.env

```diff
@@ -1 +1,4 @@
 PORT=3001
+
+REACT_APP_BACKEND_HOST="http://localhost:3000"
+REACT_APP_BACKEND_VERSION="> 0.0.0-90"
```

Comment:
I don't think that the backend version requirement should be part of the `.env` file. It is a requirement that *the code* makes, not one that *the server* makes. Maybe we should keep a `config.js` for things like this and have an additional `.env` file for things like the backend host address.